.env.example

```diff
@@ -31,9 +31,11 @@ REGISTER_TEMPLATE = mb-votingapp-signup
 REGISTER_EMAIL_TAG = votingapp_signup
 REGISTER_MB_SOURCE = votingapp
 
-MAILCHIMP_GROUP_ID = 55555
-MAILCHIMP_GROUP_NAME = VotingApp
 MC_OPT_IN_PATH = 555555
```

Comment:
See my comment below about:

- `MOBILE_OPT_IN_PATH_US`
- `MOBILE_OPT_IN_PATH_MX`
- `MOBILE_OPT_IN_PATH_BR`

@classicfreddie will provide the values.